app/src/main/java/com/android/streetworkapp/ui/navigation/TopAppBarManager.kt

```diff
@@ -0,0 +1,35 @@
+package com.android.streetworkapp.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.ui.graphics.vector.ImageVector
+
+class TopAppBarManager(
+    private var title: String = "",
+    private var hasNavigationIcon: Boolean = false,
+    private var navigationIcon: ImageVector? = null
+) {
+
+  companion object {
+    val DEFAULT_TOP_APP_BAR_NAVIGATION_ICON = Icons.AutoMirrored.Filled.ArrowBack
+  }
+  /** Changes the TopAppBar title */
+  fun setTopAppBarTitle(newTitle: String) {
+    this.title = newTitle
+  }
+
+  /** Get the TopAppBar title */
+  fun getTopAppBarTitle(): String {
+    return this.title
+  }
```

Comment:
Suggestion: You could change this getter with a more Kotlin way to get the `title` 

```kotlin
val topAppBarTitle: String
        get() = title
```       
This Kotlin "getter" could be used like this:
```kotlin
val topAppBarManager = //init your top bar manager
val title = topAppBarManager.topAppBarTitle
```

Reply:
No problem, resolved